src/lib/swapper/swappers/CowSwapper/cowExecuteTrade/cowExecuteTrade.test.ts

```diff
@@ -155,10 +192,20 @@ const expectedFoxToEthOrderToSign: CowSwapOrder = {
   quoteId: '1',
 }
 
-const deps: CowSwapperDeps = {
-  apiUrl: 'https://api.cow.fi/mainnet/api',
-  adapter: ethereumMock.ChainAdapter as unknown as ethereum.ChainAdapter,
-  web3: {} as Web3,
+const expectedUsdcToXdaiOrderToSign: CowSwapOrder = {
+  sellToken: '0xddafbb505ad214d7b80b1f830fccc89b60fb7a83',
+  buyToken: '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE',
```

Comment:
Can use `COW_SWAP_ETH_MARKER_ADDRESS` if you feel like it

Reply:
https://github.com/shapeshift/web/pull/4595/commits/66611cd028621ddf2b118660f19c70ca0cd29887